prediction/tensorflow/README.md

```diff
@@ -7,11 +7,51 @@ TensorFlow trained model in AI Platform.
     `aiplatform-submit-train-job.sh` or `local-train.sh` scripts.
 *   Run `aiplatform-deploy-model.sh`
 
+## GPU support
+
+Now if you want to deploy a new model and use GPUs, now is as simple as 
```

Comment:
Nit: Drop "Now", since down the line this wont be a new feature. 

```
If you want to deploy a new model and use GPUs, simply:
- Define the machine type
- Select the accelerator
```

Reply:
Super Nit pick (sorry) on ordering. You say define the machine type first, but your instructions select the accelerator first. Mind rearranging?

Reply:
Np, good feedback